pkg/cli/certs/trust_windows.go

```diff
@@ -0,0 +1,30 @@
+package certs
+
+import (
+	"fmt"
+
+	"github.com/magefile/mage/sh"
+	"github.com/pkg/errors"
+)
+
+const certStore = `Cert:\CurrentUser\Root`
+
+func AddTrustedCert(certPath string) error {
+	importCmd := fmt.Sprintf("import-certificate -Filepath %s -CertStoreLocation %s", certPath, certStore)
+
+	if err := sh.RunV("powershell", "-Command", importCmd); err != nil {
```

Comment:
After PowerShell 5.1+ the executable name is `pwsh.exe` 

https://learn.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell?view=powershell-7.4#powershell-executable-changes

Reply:
![image](https://github.com/aserto-dev/topaz/assets/6704579/0f855553-c4a7-4aa6-bdcf-ab584898ad34)

I'll make a couple of changes here so it works as expected on newer versions also